components/dm/spec/alertmanager.go

```diff
@@ -131,6 +131,15 @@ func (i *AlertManagerInstance) InitConfig(e executor.Executor, clusterName, clus
 		return err
 	}
 
+	// If the user specific a local config file, we should overwrite the default one with it
+	if spec.ConfigFilePath != "" {
```

Comment:
What we need to do if the path is a relative path?

Reply:
Relative path should not be allowed, I will add a check for this

Reply:
I haven't seen that it's been checked anywhere.

Reply:
I WILL do it. I mean thank for your remind and I'm doing this now.